addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -325,6 +325,9 @@ export class ColorPalette extends Component {
                 defaultColor = 'rgba(0, 0, 0, ' + this.props.opacity + ')';
             }
             this.state.customDefaultColor = defaultColor;
+        } else {
+            // Remove the colorpicker widget if "custom" is excluded.
+            this.sections['custom-colors'].parentElement.querySelector(".o_colorpicker_widget").remove();
```

Comment:
Commit message says:

> the color picker is inside the 'web_editor.ColorPalette' XML without any condition to display it

Why can't a condition be added on the XML then? Won't it be prettier?

Reply:
> Why can't a condition be added on the XML then? Won't it be prettier?

Indeed, thanks. I made the change